web/components/profile/sidebar.tsx

```diff
@@ -35,7 +38,6 @@ export const ProfileSidebar = observer(() => {
   );
 
   // Create a date object for the current time in the specified timezone
-  const currentTime = new Date();
```

Comment:
Since this is only being used here, 
`<span>
          {timeString} <span className="text-custom-text-200">{userProjectsData?.user_data.user_timezone}</span>
        </span>`

Can you please move this to a different component? or else the entire side bar will be re rendered every second